js/front/dataset/index.js

```diff
@@ -141,6 +148,35 @@ new Vue({
         },
 
         /**
+         * Get a cached checked result from extras if fresh enough
+         * @param  {Object} resource_el A resource element from DOM
+         */
+        getCachedCheck(resource_el) {
+            const extras = JSON.parse(resource_el.dataset.extras);
```

Comment:
I think to reduce JSON parsing (ie. more than 200 on SIRENE), it would be better to parse once from json-ld (and so also reuse the already in place mecanism)